### PR TITLE
UXA Fixes for #14383

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#14383

*Description of changes:*
- Clear X is now blue
- Updated text color for not found display
- Updated bold text color, resized bullet points
- A few text fixes
- Modified hairlines and spacing

image

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by dimak1** [comment]
@owillborn thanks for screenshot 👍🏻

**Review by dimak1 — APPROVED** [review]

File: ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue [hunk 1]
```diff
@@ -600,13 +600,24 @@ export default defineComponent({
...
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
+::v-deep .theme--light.v-icon.mdi-close {
+  color: $primary-blue !important;
+}
+
+.solid {
+  border: 0;
+  border-top: 0.25px solid $gray4 !important;
+}
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue:610]: The border top works for sure, just know you have options with `v-divider` to do a similar thing with more flexibility.

File: ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue [hunk 2]
```diff
@@ -600,13 +600,24 @@ export default defineComponent({
...
 
 u {
     border-bottom: 1px dotted #000;
     text-decoration: none;
 }
 
 #addHomeOwnerForm {
+  .gray {
+    color: $gray7 !important;
+  }
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue]: Perhaps follow the same approach done with the class `.gray6` in base.scss. 
ie create a global class similar, just might make your life easier in the long run, while also specifying the gray type.

**Review by cameron-eyds — APPROVED** [review]
Looks good, just some minor comments.

File: ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue]: Looks like `gray7` already exists locally in some components, so there is definitely a use case for this globally (in base.scss). 
Could you also refactor those local versions of the class as well, see HomeOwnersReview and TransferDetailsReview

**Review by ChasBoucher — APPROVED** [review]
Other than Cameron's coments.  Looks good..

File: ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue (hunk 2 above)
Comment (owillborn) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue]: Good idea, will do

**Review by doug-lovett — APPROVED** [review]
Looks good.

File: ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue (hunk 2 above)
Comment (owillborn) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue]: Looks like its just HomeOwnersReview and TransferDetailsReview that use it, changed for those as well

Comment (cameron-eyds) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue]: Right on, thanks!

**Pull request merged**